config-generators/mssql-commands.txt

```diff
@@ -43,6 +43,7 @@ add MappedBookmarks --config "dab-config.MsSql.json" --source "mappedbookmarks"
 add FteData --config "dab-config.MsSql.json" --source fte_data --permissions "anonymous:*" --graphql "FteData:FteData"
 add InternData --config "dab-config.MsSql.json" --source intern_data --permissions "anonymous:*" --graphql "InternData:InternData"
 add BooksSold --config "dab-config.MsSql.json" --source "books_sold" --rest true --graphql "books_sold:books_sold" --permissions "anonymous:*"
+add DefaultBuiltInFunction --config "dab-config.MsSql.json" --source "default_with_function_table" --rest true --graphql true --permissions "anonymous:*"
```

Comment:
Do we really need a new entity? Can this default function be a new column of an existing entity/table pair?

Reply:
That is a very good point. I went through the existing entities, all of them have specific use cases, adding some columns to fit in our use case seemed odd for them, like for a column with default function as string, or default string with paranthesis.

But keeping this point in mind we can be more cognizant from here.